demos/progress/js/progressDemo.js

```diff
@@ -84,7 +84,7 @@ var demo = demo || {};
         strings: {
             confirmStatus: "Confirm and submit your order.",
             orderSubmitted: "Order Submitted. Demo finished.",
-            percentCompleted: "%percent% Complete",
+            percentCompleted: "%percentComplete% Complete",
```

Comment:
In the string text, might be less error-pron to use `%percentCompleted%` to match up with the key.